### PR TITLE
UI/UX Improvement of Signup/login Page

### PR DESCRIPTION
**Pull request by hs05june**
#514

### CODE REVIEW

**Comment by Sarfraz-droid** [comment]
@hs05june can you place add a screenshot or gif of your changes?

File: .env.sample [hunk 1]
```diff
@@ -1,4 +1,3 @@
-
```
Comment (Sarfraz-droid) [inline .env.sample:1]: remove changes from this file

File: firebase.json [hunk 2]
```diff
@@ -17,15 +17,13 @@
     "functions": {
       "port": 5001,
       "host": "127.0.0.1"
-
```
Comment (Sarfraz-droid) [inline firebase.json:20]: avoid any changes in the root files, (except when its the main issue)
So remove any changes in `firebase.json` , `.env.sample` or `package.json`

File: src/config/index.js [hunk 3]
```diff
@@ -19,6 +19,19 @@ const firebaseConfig = {
   measurementId: process.env.REACT_APP_FIREBASE_MEASUREMENTID,
 };
```
Comment (Sarfraz-droid) [inline src/config/index.js:21]: remove changes in this config file as well, as this is not a part of the issue

**Review by Sarfraz-droid — CHANGES_REQUESTED** [review]

**Comment by hs05june** [comment]
> 

[Screencast from 05-12-22 09:45:04 PM IST.webm](https://user-images.githubusercontent.com/103871275/205818341-019e4ab6-5c30-4fed-bcac-5e5ce0cb87ce.webm)
Done the suggested changes

**Review by Sarfraz-droid — APPROVED** [review]

File: src/components/AuthPage/SignUp/index.js [hunk 4]
```diff
@@ -35,7 +41,7 @@ const SignUp = ({ background = "white" }) => {
           <Grid item={true} sm={12} className="center">
             Already have a{" "}
             <span className="brand-font text-bold">CodeLabz</span> account?{" "}
-            <Link to="/login">Log In</Link>
+            <Link to="/login"><span style={{color:'#03AAFA'}} onMouseEnter={(e)=>{handleMouseEnter(e)}} onMouseLeave={(e)=>{handleMouseLeave(e)}}>Log In</span></Link>
```
Comment (Sarfraz-droid) [inline src/components/AuthPage/SignUp/index.js:44]: Format your changes by using some formatter like prettier

**Review by Sarfraz-droid — CHANGES_REQUESTED** [review]

**Review by Maahi10001 — APPROVED** [review]

**Pull request merged**